reset-account-nonce-script/scripts/deleteAccount.js

```diff
@@ -0,0 +1,57 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+const hre = require('hardhat');
+const { Client, AccountDeleteTransaction, PrivateKey } = require('@hashgraph/sdk');
+
+async function main() {
+  const NETWORK = hre.config.defaultNetwork;
+  const MIRROR_NODE_URL = hre.config.networks[NETWORK].mirrorNodeREST;
+
+  console.log(`Selected network: ${NETWORK}`);
+
+  const deletableWallet = new hre.ethers.Wallet(process.env.DELETABLE_ACCOUNT_PK);
+
+  const preDeletionAccount = await (await fetch(`${MIRROR_NODE_URL}/api/v1/accounts/${deletableWallet.address}`)).json();
+  console.log(`address: ${preDeletionAccount.evm_address} deleted: ${preDeletionAccount.deleted} balance: ${preDeletionAccount.balance.balance}`);
+
+  const client = Client.forNetwork(NETWORK)
+    .setOperator(process.env.OPERATOR_ID, PrivateKey.fromStringECDSA(process.env.OPERATOR_PK));
```

Comment:
Here's where I'd say use a value calculated from the evm address instead of asking them to provide an ID.

Basically call the mirror node and get the value

Reply:
Thanks for the clarification 👍